data-structures/hashmap.md

```diff
@@ -290,7 +290,7 @@ As yet, there is no known hash collision attack vector against IPLD data structu
 
 The [Filecoin Project blockchain](https://filecoin-project.github.io/specs/) makes use of a HAMT that uses the same HAMT with buckets and CHAMP mutation semantics as outlined in this document. It encodes directly as [DAG-CBOR](../block-layer/codecs/dag-cbor.md) but uses a different block layout to the one specified here. This section documents the specific ways that the Filecoin HAMT variant differs from this specification. IPLD HashMap implementations may be able to implement a form that provides compatibility with Filecoin when requested by the user.
 
-The reference Go implementation for the Filecoin HAMT is used by the [Lotus](https://lotu.sh/) client and is available at <https://github.com/ipfs/go-hamt-ipld>.
+The reference Go implementation for the Filecoin HAMT is used by the [Lotus](https://lotu.sh/) client and is available at <https://github.com/filecoin-project/go-hamt-ipld>. Note that this document reflects v3 of the Go module, whose latest version is `v3.0.1` at the time of writing.
```

Comment:
did we actually get it updated for the v3 form? I don't even remember now

Reply:
You're absolutely right, I did not actually do that :facepalm:

This is what happens when I pick up a months-old branch and I forget that I hadn't finished it. Element was changed from keyed to kinded for v3, for example.